cli/src/commands/list.ts

```diff
@@ -30,6 +30,12 @@ export const listFilesCommand = new Command('list')
     const metadatas = await getGlobalMetadata(provider, webSiteAddress)
     const webSiteDefaultMetadata = extractWebsiteMetadata(metadatas)
 
+    const isimmutable = await isImmutable(
+      webSiteAddress,
+      globalOptions.node_url,
+      true // wait final execution because it's a critical operation
+    )
```

Comment:
In fact, this could be improved by checking both final and candidate. So if the code final is not empty, but candidate is, then the website might soon be immutable and so we could show that it's in the process of becoming immutable (ATM, finality is like 30 sec, but AFAIK it can take more time depending of the chain usage)

But I don't think it's mandatory for now, maybe we should just open an issue about it